dbtwiz/dbt/manifest.py

```diff
@@ -8,8 +8,10 @@
 
 from dbtwiz.config.project import project_config, project_dbtwiz_path
 from dbtwiz.config.user import user_config
+from dbtwiz.config.theme import Theme
```

Comment:
Should run `isort` to correct the import order in all changed files. 
E.g for me this would be `isort C:\Users\sindr\Documents\repos\amedia\dbtwiz\dbtwiz`.

Should also run `ruff format` for changed files.